src/python/espressomd/constraints.py

```diff
@@ -420,21 +431,20 @@ class LinearElectricPotential(Constraint):
     """
     Electric potential of the form
 
-    :math:`\\phi = -E \\cdot x + \\phi_0`,
+    :math:`\\phi = -\\vec{E} \\cdot \\vec{r} + \\phi_0`,
```

Comment:
Actually I'm not a big fan of this change. For me $\vec{r}$ is a distance vector, whereas $\vec{x}$ is the absolute position. And since this is one of the few places the absolute position is relevant, I think the documentation should reflect that.

Reply:
I was not entirely sure about the notation. Quantum mechanics uses $|x\rangle$ and $\hat{x}$ ([position operator](https://en.wikipedia.org/wiki/Position_operator)), while classical mechanics seems to accept both $\vec{x}(t)$ and $\vec{r}(t)$ ([Newton equations](https://en.wikipedia.org/wiki/Classical_mechanics#Velocity_and_speed) and [geometry](https://en.wikipedia.org/wiki/Position_(geometry))). What confused me is how inconsistent the user guide is about notations. For example in chapter [Integrators and thermostats](https://espressomd.github.io/doc/integration.html), we use $\vec{x}(t)$ in VV, NpT and $\vec{r}(t)$ in steepest descent, Brownian. There is often no indication that $x$ is a vector (no arrow, no boldface), so I'm not even sure if the author meant the position vector or the x-component of the position vector.